### PR TITLE
Remove content-ui

### PR DESCRIPTION
**Pull request by Wawrzyn321**
**Description**

Changes proposed in this pull request:

- Remove `clusterAssetGroup` query
- Remove `content-ui` chart
- Remove Rafter from Console smoke test config

**Related issue(s)**
[Original issue](https://github.com/kyma-project/console/issues/2056)

### CODE REVIEW

File: components/console-backend-service/internal/domain/schema.resolvers.go [hunk 1]
```diff
@@ -677,3 +673,13 @@ type serviceClassResolver struct{ *Resolver }
 type serviceInstanceResolver struct{ *Resolver }
 type servicePlanResolver struct{ *Resolver }
 type subscriptionResolver struct{ *Resolver }
+
+// !!! WARNING !!!
+// The code below was going to be deleted when updating resolvers. It has been copied here so you have
+// one last chance to move it out of harms way if you want. There are two reasons this happens:
+//  - When renaming or deleting a resolver the old code will be put in here. You can safely delete
```
Comment (dariadomagala) [inline components/console-backend-service/internal/domain/schema.resolvers.go:680]: can we remove it then? :)

**Review by dariadomagala — APPROVED** [review]

**Review by kazydek — DISMISSED** [review]

File: resources/core/values.yaml [hunk 2]
```diff
@@ -17,11 +17,6 @@ global:
   api_controller:
     dir:
     version: 3b88aaaf
-  docs:
-    # - Change the value below to the branch from your fork if you want to point to documentation sources from your fork
-    # - Change the value below to the release branch during the release
-    # Example: clusterAssetGroupsVersion: "release-0.9"
-    clusterAssetGroupsVersion: master
```
Comment (aerfio) [inline resources/core/values.yaml:24]: This implies that there are changes needed to release process

**Review by aerfio — CHANGES_REQUESTED** [review]
Whoops, with those changes the website is not working, we need to investigate this

File: resources/core/values.yaml [hunk 3]
```diff
@@ -18,6 +18,7 @@ global:
     dir:
     version: 3b88aaaf
   docs:
+    enabled: false
```
Comment (aerfio) [inline resources/core/values.yaml:21]: right now it's `global.docs.enabled`, you have to either change this file or requirements.yaml in order to install kyma without those clusterassetgroups

**Review by aerfio — CHANGES_REQUESTED** [review]

**Review by aerfio — APPROVED** [review]

**Review by akucharska — APPROVED** [review]

**Review by kazydek — APPROVED** [review]

**Review by pbochynski — APPROVED** [review]

**Review by dariadomagala — APPROVED** [review]

**Pull request merged**